modules/dmrpp_module/DmrppArray.cc

```diff
@@ -1346,7 +1346,7 @@ void DmrppArray::read_linked_blocks(){
         throw BESInternalError("The number of linked blocks must be >1 to read the data.", __FILE__, __LINE__);
 
     //Change to the total storage buffer size to just the compressed buffer size.
-    reserve_value_capacity_ll_byte(get_var_chunks_storage_size());
+    //reserve_value_capacity_ll_byte(get_var_chunks_storage_size());
```

Comment:
Sonar will probably ask for the commented out code to be removed.

Reply:
Have another pull request, I made a note to remove this in that pull request.